kahpp-spring-autoconfigure/src/test/java/dev/vox/platform/kahpp/unit/configuration/http/SimpleHttpCallTest.java

```diff
@@ -135,4 +138,60 @@ private JsonNode getJsonNodeFromFile(String file) throws IOException {
             Paths.get("src/test/resources/Fixtures/collection/collection_6/" + file + ".json"));
     return new ObjectMapper().readTree(keyString);
   }
+
+  @Test
+  void shouldReplacePathWithValue() throws IOException {
+
+    httpCall =
+            new SimpleHttpCall(
+                    "http_test",
+                    Map.of(
+                            "api",
+                            "suchNiceApi",
+                            "path",
+                            HTTP_CALL_PATH+"/${value.payload.id}",
+                            "apiClient",
+                            apiClient,
+                            "responseHandler",
+                            ResponseHandlerRecordUpdate.RECORD_VALUE_REPLACE));
+
+    JsonNode key = getJsonNodeFromFile(FIXTURE_KEY);
+    JsonNode value = getJsonNodeFromFile(FIXTURE_VALUE);
+
+    KaHPPMockServer.mockHttpResponse(HTTP_CALL_PATH+"/"+value.get("payload").get("id").textValue(), value.toString(), 200, "{}");
```

Comment:
Minor thing: as I can see from the source code around spaces are used in string concatenation expressions, maybe we can add some automatic code autoformatting to deal with such cases.

Reply:
Let's discuss it tomorrow morning; I didn't understand your point.